### ISSUE TITLE
Store example tile images in a directory under doc/

### ISSUE THREAD

**Issue by hryniuk** [Wangscape/Wangscape#134]
For now, each example under `doc/` directory has its own tile images. Some of them (e.g. `grass.png` and `sea.png`) are duplicated and adding new examples with the same images under different directories may needlessly affect a size of `Wangscape` repository.

We could store them in one directory, e.g. `doc/example_images`, but it will affect on running these examples - one won't be able to just move example to different directory (e.g. `build/`) and run them, it will require either copying images or changing file paths in options JSON.

We should decide, what's more important: a bit faster `fetch`/`push` or the convenience of checking examples provided with Wangscape.

**Comment by serin-delaunay** [comment]
I think there are good reasons for and against having copies of base tiles in the repository:
* They're small (<3K), few, and not often updated, so they're not affecting git command times much.
* Keeping them self-contained means that a user can copy a single example for editing anywhere without breaking it.
* Keeping multiple copies in the repository means that if they're updated (as #87 requires), they must be updated individually.
* When a larger variety of examples is added, keeping copies may start to slow down git commands.

There are some options other than putting them all in a folder and updating the relative paths in the examples, for example:
* Put the examples in a separate (potentially slow) repository.
* Put them in one folder which is intended to be copied into an example folder before use? Not sure about this one.
* Wangscape is a kind of compiler; source tile images and module group JSON files are its libraries. Give it a way to retrieve these from both a central store and from paths relative to the configuration file.

I like the last option, but it would require some time to implement and very careful design (it should still be accessible to non-programmers and portable across supported platforms).

**Comment by hryniuk** [comment]
Some thoughts:
* as [Wangscape repository](https://github.com/Wangscape/Wangscape) is used for development and users should get Wangscape from Releases page or their distribution's repository, in my opinion, there's no need to keep there more than, hmm... four examples
* we could add more examples to a package/archive, when releasing a new version
* by keeping a separate repository with examples we won't be limited by it's size and that could encourage contributions (moreover some users may feel overwhelmed by size of Wangscape repository and all these guidelines in `*.md` files)

**Comment by serin-delaunay** [comment]
Yes, perhaps it would be best to have two separate groups of examples:
* One small group kept in the dev repository, designed to employ as many Wangscape features as possible and used for testing. Base tiles have only one copy stored, this makes relative paths in config files awkward, and that's OK. Major new features may mean a new example is added, if necessary.
* A separate repository with just example configurations, designed as aesthetic demonstrations, template projects, or tutorials. Each example can be self-contained, with copies of common tiles allowed.
